Regional historical changes for REF (#3926)
ESMValGroup/ESMValTool: doc/sphinx/source/recipes/recipe_ref.rst

Available recipes
-----------------

> Recipes are stored in `recipes/ref`

Reviewer: ```suggestion
Recipes are stored in `recipes/`
```

* :ref:`recipe_ecs.yml <recipes_ecs>`:
  Calculate equilibrium climate sensitivity (ECS)
* :ref:`recipe_tcr.yml <recipes_tcr>`:
  Calculate transient climate response (TCR)
* :ref:`recipe_tcre.yml <recipes_tcre>`:
  Calculate transient climate response to cumulative CO2 emissions (TCRE)

> * recipe_ref_cre.yml:

Reviewer: ```suggestion
* ref/recipe_ref_cre.yml:
```

  Maps and zonal means of longwave and shortwave cloud radiative effect
* ref/recipe_model_benchmarking_timeseries_region.yml:

>   Time series in comparison with refernce data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_timeseries.yml <recipe_benchmarking>`

Reviewer: ```suggestion
  Time series in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_timeseries.yml <recipe_benchmarking>`
```

* ref/recipe_model_benchmarking_annual_cycle_region.yml:

>   Annual cycle in comparison with refernce data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_annual_cycle.yml <recipe_benchmarking>`

Reviewer: ```suggestion
  Annual cycle in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_annual_cycle.yml <recipe_benchmarking>`
```

* ref/recipe_seaborn_trend_regions.yml:

>   Linear Trends for all IPCC regions compaerd with refernce data, based on :ref:`recipe_seaborn.yml <recipes_seaborn_diag>`

Reviewer: ```suggestion
  Linear Trends for all IPCC regions compared with reference data, based on :ref:`recipe_seaborn.yml <recipes_seaborn_diag>`
```

  Calculate transient climate response to cumulative CO2 emissions (TCRE)
* ref/recipe_ref_cre.yml:
  Maps and zonal means of longwave and shortwave cloud radiative effect
* ref/recipe_model_benchmarking_timeseries_region.yml:
  Time series in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_timeseries.yml <recipe_benchmarking>`
* ref/recipe_model_benchmarking_annual_cycle_region.yml:
  Annual cycle in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_annual_cycle.yml <recipe_benchmarking>`
* ref/recipe_seaborn_trend_regions.yml:

>   Linear Trends for all IPCC regions compared with reference data, based on :ref:`recipe_seaborn.yml <recipes_seaborn_diag>`

Reviewer 2: Here I would just use one region, as you do for the other two recipes. Then you can adjust the text to be as general as the texts in the description of the other recipes before.

   Zonal mean vertically resolved ozone climatology from CNRM-ESM2-1 compared with ESACCI-OZONE
   for the years 1990 to 2000. Created with recipe_ref_ozone.yml.

.. _fig_ref_7:
.. figure::  /recipes/figures/ref/timeseries_tas_ambiguous_dataset_Amon_historical_r1i1p1f1.png
   :align:   center
   :width:   8cm

   Time series of near-surface air temperature anomalies from MIROC6 compared with HadCRUT5

>    for N.Europe for the years 1980 to 2014 (refernce period 1980 to 2009). Created with recipe_model_benchmarking_timeseries_region.yml.

Reviewer 2: There is a typo in the word "reference".

Author: Thanks, I removed it
